dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/RpcContext.java

```diff
@@ -144,32 +143,6 @@ public static void restoreContext(RpcContext oldContext) {
         LOCAL.set(oldContext);
     }
 
-
-    public RpcContext copyOf() {

```

Comment:
This method will be invoked by the code that is implements in dubbo-2.6.x

Reply:
Why will this method being used in 2.6.x?